File: codechef/DSTAPLS.py
# https://www.codechef.com/AUG19B/problems/DSTAPLS
# Distribute Apples

try:
    T = int(input())
    while T!=0:
        T=T-1
        N,K = [int(x) for x in input().split()]
        if N%K**2 == 0:
            print("NO")
        else:
            print("YES")
except EOFError as e:
    print(end="")

File: codechef/DSTAPLS.c
// All test cases passed
// https://www.codechef.com/AUG19B/problems/DSTAPLS
// Distribute Apples

#include<stdio.h>

int main(){
    int T;
    unsigned long long N,K,m;
    scanf("%d",&T);
    while(T--){
        scanf("%llu%llu",&N,&K);
        // printf("%llu\n%llu\n",N,K);
        if(K!=0){
        m = N/K;
        if(m%K == 0)
            printf("NO\n");
        else
            printf("YES\n");
        }      
    }
    return 0;
}